js/src/redux/reducers/dismissedAlerts.js

```diff
@@ -0,0 +1,33 @@
+import { DISMISS_ALERT } from "../actions/dismissedAlerts";
+
+const INITIAL_STATE = window.wpseoScriptData.dismissedAlerts;
```

Comment:
Completely agree with the safety check, will add that. Keeping the direct call of the `window` for now, since using `populateStore` would probably result in a temporary displaying of a dismissed alert until that dispatch is processed.